packages/theme-chalk/src/drawer.scss

```diff
@@ -103,6 +104,9 @@ $directions: rtl, ltr, ttb, btt;
       }
     }
   }
+  > .#{$panel} {
```

Comment:
Perhaps we should add relevant styles during the opening and closing transitions to avoid causing other effects.